setup/fio-setup-basic

```diff
@@ -189,6 +189,16 @@ for storage in $storages; do
 		dev=/dev/$(basename $storage)
 		if is_null_blk $dev; then
 			storage_setup="filename=/dev/nullb0"
+		elif is_mount_on_root $dev; then
+			[ -d "/lkp/benchmarks/fio" ] || mkdir /lkp/benchmarks/fio
```

Comment:
refer to comment for early commit, here i think we can do below for now to keep  one series of if/elif clause.

elif is_mount_on_root /dev/$(basename $storage)

Reply:
thanks for the feedback, i have made the requested changes